tailwind.config.js

```diff
@@ -133,6 +133,9 @@ module.exports = {
         96: "24rem",
         128: "32rem",
       },
+      width: {
+        64: "19rem",
```

Comment:
Why are writing custom width, Can't we take the closet available in Tailwind ?

Reply:
I just removed it thank you